doc/source/rllib-examples.rst

```diff
@@ -36,12 +36,17 @@ Training Workflows
 Custom Envs and Models
 ----------------------
 
+- `Local Unity3D multi-agent environment example <https://github.com/ray-project/ray/tree/master/rllib/examples/unity3d_env_local.py>`__:
+   Example of how to setup an RLlib Trainer against a locally running Unity3D editor instance to
+   learn any Unity3D game (including support for multi-agent).
+   Use this example to try things out and watch the game and the learning progress live in the editor.
+   For a more heavy-weight, distributed, cloud-based example, see `Unity3D client/server`_ below.
```

Comment:
Can we clarify that the client/server isn't really needed for distributed training? (not sure about the wording here, maybe we can consider removing the example, though it does seem useful for other possible scenarios)

Reply:
Yes. I think it's still a very useful example. We don't really know, how people would use Unity3D. In this simple case here, RLlib has full control over all agents, but you could imagine having a game server (the PolicyClient in our case) with external players, in which case control should really happen from within the engine, not RLlib anymore.